app/src/main/res/values/strings.xml

```diff
@@ -375,7 +375,7 @@
     <string name="installation_aborted_description">The installation was cancelled manually. Try again?</string>
     <string name="installation_blocked_description">The installation was blocked. Review your device security settings and try again.</string>
     <string name="installation_conflict_description">The installation was prevented by an existing installation of the app. Uninstall the installed app and try again?</string>
-    <string name="installation_incompatible_description">The app is incompatible with this device. Contact the developer of the app and ask for support.</string>
+    <string name="installation_incompatible_description">The APK is incompatible with this device. Use the correct APK for your device and try again.</string>
```

Comment:
I think "The APK" at the beginning should be remain "The app".
Because it's inconsistent with other messages and it is talking about the patched **app** here.